### ISSUE TITLE
Alpine. Pygraphviz 1.4rc1 - failed to find graphviz

### ISSUE THREAD

**Issue by tigrus** [pygraphviz/pygraphviz#141]
When trying to install `pygraphviz 1.4rc1` on alpine - getting this error. `1.3` installs fine. 
```
  Your Graphviz installation could not be found.

          1) You don't have Graphviz installed:
             Install Graphviz (http://graphviz.org)

          2) Your Graphviz package might incomplete.
             Install the binary development subpackage (e.g. libgraphviz-dev or similar.)

          3) You are using Windows
             There are no PyGraphviz binary packages for Windows but you might be
             able to build it from this source.  See
             http://networkx.lanl.gov/pygraphviz/reference/faq.html

          If you think your installation is correct you will need to manually
          provide path to graphviz include and library. For example:

          pip install pygraphviz --install-option="--include-path=/usr/include/graphviz" --install-option="--library-path=/usr/lib/graphviz/"

          The current setting of library_dirs and include_dirs is:
  library_dirs=None
  include_dirs=/usr/include/graphviz

  error: Error locating graphviz.
```

Dockerfile to reproduce: 
```
FROM python:3.6.3-alpine

ADD requirements.txt /app/requirements.txt

RUN apk update && \
    apk add --no-cache libstdc++ graphviz postgresql-libs && \
    apk add --no-cache --virtual .build-deps python-dev g++ make swig linux-headers postgresql-dev graphviz-dev && \
    pip3 install -r /app/requirements.txt && \
    apk del .build-deps
```

requirements.txt: 
```
pygraphviz==1.4rc1
```

**Comment by tycho01** [comment]
related: https://stackoverflow.com/questions/15661384/python-does-not-see-pygraphviz

**Comment by albertjan** [comment]
```
apk add pkgconfig graphviz graphviz-dev gcc musl-dev && \
pip3 install pygraphviz --install-option="--library-path=/usr/lib/graphviz/"
```

this did the trick for me. some how pgk-tool doesn't find the library dir so you have to pass it yourself

**Comment by rossbar** [comment]
This is hopefully resolved in pygraphviz 1.7 which was just released. I'm not familiar with `apk` specifically, but if Graphviz headers and libs are installed in `/usr/include` and `/usr/lib/graphviz` respectively, you should no longer need to explicitly specify include and lib paths with `pip`. See the [updated install instructions](https://pygraphviz.github.io/documentation/stable/install.html#linux) for the recommended procedure on some common distros.

If the equivalent doesn't work on Alpine w/ `apk` however, please reopen this or open a new issue!
